src/modules/profile/pages/profileSettings/components/CreatorNotifications.tsx

```diff
@@ -0,0 +1,180 @@
+import { useMutation, useQuery } from '@apollo/client'
+import { Avatar, FormControl, FormLabel, HStack, Select, Switch, VStack } from '@chakra-ui/react'
+import { useState } from 'react'
+import { useParams } from 'react-router'
+
+import { MUTATION_UPDATE_CREATOR_NOTIFICATIONS_SETTINGS } from '@/modules/profile/graphql/mutations/creatorNotificationMutation'
+import { QUERY_CREATOR_NOTIFICATIONS_SETTINGS } from '@/modules/profile/graphql/queries/creatorNotificationSettingsQuery'
+import { Body } from '@/shared/components/typography'
+
+type NotificationConfiguration = {
+  id: string
+  name: string
+  value: string
+  type: string
+  options?: string[]
+}
+
+type NotificationSettings = {
+  notificationType: string
+  isEnabled: boolean
+  configurations: NotificationConfiguration[]
+}
+
+export const CreatorNotifications = () => {
+  const { userId } = useParams()
+  const [notificationSettings, setNotificationSettings] = useState<NotificationSettings[]>([])
+  const [project, setProject] = useState<any>(null)
```

Comment:
why is this any, you could use a Fragment for this one.

Reply:
I forgot about it but it was not relevant anyways because this should display multiple project notifications